README.md

```diff
@@ -62,11 +62,7 @@ A router in HNDL is defined using the function `router`, it takes a variadic lis
 endpoints like this:
 
 ```ts
-const myRouter = router(
-  firstEndpoint,
-  secondEndpoint,
-  thirdEndpoint
-)
+const myRouter = router(firstEndpoint, secondEndpoint, thirdEndpoint);
```

Comment:
I would suggest the formatting that was before, it's a bit more readable and easy to understand the kind of structure we have.

Reply:
yes, you're right, and that's my prettier format than who did this and I didn't notice. my apologies.